ruby-test-mode.el

```diff
@@ -43,6 +43,7 @@
 ;; C-c C-s   - Toggle between implementation and test/example files.
 
 (require 'ruby-mode)
+(require 'pcre2el)
```

Comment:
I think this needs to be added to the Cask file and to the `Package-Requires` section as well.